pingouin/multivariate.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pandas as pd
+import scipy
```

Comment:
Please avoid generic call to scipy and instead import the specific function (from scipy.stats import chi2)